app/models/katello/concerns/content_aspect_host_extensions.rb

```diff
@@ -0,0 +1,22 @@
+module Katello
+  module Concerns
+    module ContentAspectHostExtensions
+      extend ActiveSupport::Concern
+
+      included do
+        has_one :content_aspect, :class_name => '::Katello::Host::ContentAspect', :foreign_key => :host_id, :inverse_of => :host, :dependent =>  :destroy
+        has_one :content_view, :through => :content_aspect
+        has_one :lifecycle_environment, :through => :content_aspect
```

Comment:
Should we note that here? Doesn't really prevent them from being used elsewhere, but perhaps we at least somewhat recall why.